modular-docs-manual/files/templates-without-comments/assembly-a-collection-of-modules.adoc

```diff
@@ -0,0 +1,32 @@
+
+ifdef::context[:parent-context: {context}]
+
+
+[id="a-collection-of-modules"]
+
+= A collection of modules
+
+:context: assembly-keyword
+
+[role="system:abstract"]
+This paragraph is the assembly introduction. It explains what the user will accomplish by working through the modules in the assembly and sets the context for the user story the assembly is based on. The `[role="system:abstract"]` tag defines the first paragraph of the introduction for search metadata.
+
+.Prerequisites (optional)
+
+* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
+* You can also link to other modules or assemblies the user must follow before starting this assembly.
+* Delete the section title and bullets if the assembly has no prerequisites.
+
+include::modules/TEMPLATE_CONCEPT_explaining_a_concept.adoc[leveloffset=+1]
+
+include::modules/TEMPLATE_PROCEDURE_doing_one_procedure.adoc[leveloffset=+1]
+
+[role="system:additional-resources"]
+== Additional resources
```

Comment:
Should this be marked (optional) too? Like the Prereqs? Not a bit deal and not what this PR is fixing, so feel free to ignore me.

Reply:
Thanks @ncbaratta. Good point. I made a few other tweaks, as well as updates to the annotated templates.